Schema/V1/arkivstruktur.xsd

```diff
@@ -228,6 +228,10 @@
             <xs:element name="partID" type="n5mdk:partID" minOccurs="0"/>
             <xs:element name="partNavn" type="n5mdk:partNavn"/>
             <xs:element name="partRolle" type="n5mdk:partRolle"/>
+            <xs:choice minOccurs="0">
```

Comment:
Flytt til slutten av objektet slik at dette ikke brekker xml

Reply:
Im a bit uncertain about this. Putting this at the end looks bad since the ordering then is not equal across services.

Reply:
Either avoid breaking changes or reorder all objects to consistent ordering.